core/src/View/MDSpan/Kokkos_MDSpan_Header.hpp

```diff
@@ -44,7 +44,39 @@ using std::layout_stride;
 using std::mdspan;
 }  // namespace Kokkos
 #else
+// Opt in for Kokkos::pair to submdspan/subview
+// submdspan does only take index_pair_like which is derived from tuple_like
```

Comment:
So `Kokkos::pair` will never work if the mdspan implementation is provided by the compiler?

Reply:
nope